onnxruntime/core/providers/webnn/allocator.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <emscripten.h>
+#include <emscripten/val.h>
+
+#include "core/common/inlined_containers.h"
+#include "core/framework/allocator.h"
+#include "core/framework/ortdevice.h"
+
+namespace onnxruntime {
+namespace webnn {
+
+class WebNNBufferAllocator : public IAllocator {
+ public:
+  WebNNBufferAllocator() : IAllocator(OrtMemoryInfo("WebNNMLBufferAllocator", OrtAllocatorType::OrtDeviceAllocator, OrtDevice(OrtDevice::GPU, OrtDevice::MemType::DEFAULT, 0), 0, OrtMemTypeDefault)) {}
```

Comment:
should use `WEBNN_BUFFER` instead of `"WebNNMLBufferAllocator"`.

Reply:
Done